### DISCUSSION TITLE
SSL cert taking a really long time to generate

### DISCUSSION THREAD

**Discussion by mozzius** [vercel/community#2498] [Help]
### Summary

Pretty much just the title. Added a new domain to a new project, and it's been adding the SSL cert for at least an hour and a half, if not longer

### Additional information

Using nameservers. The domain is working, albeit without SSL

**Comment by smaeda-ks** [comment]
Hi @mozzius, which domain is this about? I can take a look

**Reply by mozzius** [reply]
swifties.social - it resolved itself shortly after posting this :)

**Reply by smaeda-ks** [reply]
No worries, glad it's resolved!
